sys/mathoid.js

```diff
@@ -114,10 +119,16 @@ class MathoidService {
                 }));
             }
             // construct the request object that will be emitted
+            Object.assign(completeBody[format].headers, {
+                'x-store-x-resource-location': hash
```

Comment:
`s/x-store-//` since `x-store-` is prepended later

Reply:
indeed.